MediaWikiTelegramBot.js

```diff
@@ -284,7 +293,38 @@ function showCurrentMwMessage(userID) {
 function advanceMwMessage(userID) {
     const user = getUser(userID);
 
+    // Make sure there is another message to translate
+    if( user.currentMwMessageIndex + 1 >= user.loadedMwMessages.length ) {
+        user.currentMwMessageIndex = 0;
+        user.loadedMwMessages = [];
```

Comment:
This could also be moved to a function, since it repeats itself in at least one more place.